mk/common.mk

```diff
@@ -38,8 +38,6 @@ getdef = $(shell sed -n "0,/$(1)/s/\#define\s\+$(1)\s\+\(\w*\)/\1/p" $(2))
 
 ifeq ($(PLATFORM),x86_64)
 getarch = $(shell [ -f /etc/debian_version ] && echo "amd64" || echo "x86_64")
-else ifeq ($(PLATFORM),ppc64le)
-getarch = $(shell [ -f /etc/debian_version ] && echo "ppc64el" || echo "ppc64le")
-else
-$(error Unsupported architecture)
+else ifeq (,$(filter $(PLATFORM),ppc64le ppc64el))
```

Comment:
i think i need to undo this, actually. i was trying to figure out what was going on with the debian build.